packages/browser-utils/src/metrics/browserMetrics.ts

```diff
@@ -300,6 +300,13 @@ interface AddPerformanceEntriesOptions {
    * sent as a standalone span instead.
    */
   recordClsOnPageloadSpan: boolean;
+
+  /**
+   * Resource spans matching strings in the array will not be emitted.
```

Comment:
Just a small suggestion to make it as clear as possible for users:

```suggestion
   * Resource spans with `op`s matching strings in the array will not be emitted.
```